python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -114,15 +114,21 @@ def _attribute_filter__set__(self, name, value):
             #   Do some checking of the filenames in a subprocess
             if name == 'lfn':
                 self.namePattern = os.path.basename(value)
-                self.remoteDir = os.path.dirname(value)
+                if os.path.dirname(value):
+                    self.remoteDir = os.path.dirname(value)
+                elif os.path.isfile(os.path.join(os.getcwd(), self.namePattern)):
+                    self.remoteDir = os.getcwd()
```

Comment:
How does this work though? If `remoteDir` is on the local file system and `localDir` is on the local file system then how is this a *DIRAC* file?